### ISSUE TITLE
SA1118 does not allow multiline tuples

### ISSUE THREAD

**Issue by DrummingBeb** [DotNetAnalyzers/StyleCopAnalyzers#3707]
It seems that it should be allowed to put the elements of tuples on separate lines like
```
private Wrapper GetTuple()
        => new(
            ("------------------------ long string -----------------------",
                "------------------------ long string -----------------------"),
            ("------------------------ long string -----------------------",
                "------------------------ long string -----------------------"));
```
However, rule SA1118 forces me to write really long lines
```
private Wrapper GetTuple()
        => new(
            ("------------------------ long string -----------------------",  "------------------------ long string -----------------------"),
            ("------------------------ long string -----------------------",  "------------------------ long string -----------------------"));
```
or to use more variables.

**Comment by sharwell** [comment]
The closest existing rule for this would (in theory) allow the following style:

```cs
new ValueTuple<string, string>(
  "------------------------ long string -----------------------",
  "------------------------ long string -----------------------")
```

This would suggest the following as the corresponding tuple style:

```cs
(
  "------------------------ long string -----------------------",
  "------------------------ long string -----------------------")
```

I'm not sure I'm ready for this format yet.

**Comment by benjiwolff** [comment]
I like the suggested format. The indentation makes it clear, that the elements belong to the tuple and are not top-level arguments themselves and the alignment is also kept if wrapped **after** opening brace. People who do not wrap after the opening brace are probably used to the jagged look where the first argument is not aligned. Either way, the jaggedness is in my opinion much less problematic than an unnecessarily long line.
